components/devtools/actors/root.rs

```diff
@@ -220,13 +232,30 @@ impl Actor for RootActor {
             },
 
             "getTab" => {
-                let tab = registry.find::<TabDescriptorActor>(&self.tabs[0]);
-                let reply = GetTabReply {
-                    from: self.name(),
-                    tab: tab.encodable(registry),
-                };
-                let _ = stream.write_json_packet(&reply);
-                ActorMessageStatus::Processed
+                if let Some(serde_json::Value::Number(browserId)) = msg.get("browserId") {
```

Comment:
Would it be possible to move this code into a function in order to use an early return pattern and avoid some indentation?

Reply:
Absolutely, how about now? I used let else to further reduce indentation but I can change it if another pattern is preferred. I am also not sure about the name `get_tab_msg_by_browser_id` but I couldn't think of one that was shorter but still descriptive, any suggestion is welcome.

Reply:
This looks great now. Thank!